src/test/resources/features/meteringLabels.feature

```diff
@@ -0,0 +1,5 @@
+@check-metering-labels
+  Feature: Check metering labels
+    Scenario: Check that metering labels have correct values
+      When check that Apicurito has 7 and they are running
```

Comment:
Probably would be better to check each resource separately to ensure that we have 3x UI pod, 3x generator pod and 1x operator pod.